adhocracy4/test/helpers.py

```diff
@@ -33,3 +37,29 @@ def redirect_target(response):
 def render_template(string, context=None):
     context = Context(context or {})
     return Template(string).render(context)
+
+
+@contextmanager
+def skip_background_mail():
```

Comment:
So it was generic for all decorators not only for the background-task decorator. My intention was to only make it generic for the background-task decorator. So it seems to me there are those options worth to discuss:

1. Move the entire thing as it is from helpers to the email test package, because it is only useful there
2. Make a generic replace decorator Y in module X with function Z. Where X = backgroundtask Y = adhcoracy4.emails and Z= a function that checks json serilisabilityof arguments, in our current use case.

Anyhow these are only mere cosmetic changes. The docstring is more important for me.

PS: Arbeiten in der Freizeit ist der Feind der Arbeiterschaft :factory: